frontend/packages/ceph-storage-plugin/integration-tests/views/pvc.view.ts

```diff
@@ -0,0 +1,95 @@
+  by.xpath(".//dt[text()='Annotations']/following-sibling::dd[1]"),
+);
+export const pvcLabelSelector = element(
+  by.xpath(".//dt[text()='Label Selector']/following-sibling::dd[1]"),
+);
+export const pvcCreatedAt = element(
+  by.xpath(".//dt[text()='Created At']/following-sibling::dd[1]"),
+);
+// in 4.2 there was no owner
+export const pvcOwner = element(by.xpath(".//dt[text()='Owner']/following-sibling::dd[1]"));
+export const pvcStatus = element(by.xpath(".//dt[text()='Status']/following-sibling::dd[1]"));
+export const pvcSize = element(by.xpath(".//dt[text()='Capacity']/following-sibling::dd[1]"));
+// in in 4.2 there 7th and 8th items were Size and Requested, in 4.3 it's Capacity
+export const pvcAccessMode = element(
+  by.xpath(".//dt[text()='Access Modes']/following-sibling::dd[1]"),
+);
+export const pvcVolumeMode = element(
+  by.xpath(".//dt[text()='Volume Mode']/following-sibling::dd[1]"),
+);
+export const pvcStorageClass = element(
+  by.xpath(".//dt[text()='Storage Class']/following-sibling::dd[1]"),
+);
+export const pvcPersistentVolume = element(
+  by.xpath(".//dt[text()='Persistent Volume']/following-sibling::dd[1]"),
+);
+export const actionsButton = $('[data-test-id=actions-menu-button]');
+export const deletePvc = $('[data-test-action="Delete Persistent Volume Claim"]');
+
+// list of PVCs
+export const nameInTable = (name) => $(`a[data-test-id=${name}]`);
+
+export const goToPersistentVolumeClaims = async () => {
+  await sideNavView.clickNavLink(['Storage', 'Persistent Volume Claims']);
+  await crudView.isLoaded();
+};
+
+export const createNewPersistentVolumeClaim = async function(
+  newPvcName: string,
+  namespace: string,
+  newPvcSize: string,
+  sizeUnits: string,
+  storageClass: string,
+  accessMode: string,
+  waitForBinding: boolean,
+) {
+  await goToPersistentVolumeClaims();
+  await selectItemFromDropdown(namespace, namespaceDropdown);
+  await click(crudView.createYAMLButton);
+  await browser.wait(
+    until.textToBePresentInElement($('.co-m-pane__heading'), 'Create Persistent Volume Claim'),
+  );
+  await browser.wait(until.and(crudView.untilNoLoadersPresent));
+  await selectItemFromDropdown(storageClass, storageclassDropdown);
+  await inputPVCName.sendKeys(newPvcName);
+  await click(selectAccessMode(accessMode));
+  await inputPVCSize.sendKeys(newPvcSize);
+  // Units should be Mi, Gi or Ti
+  await selectItemFromDropdown(sizeUnits, sizeUnitsDropdown);
+  await click(crudView.saveChangesBtn);
+  if (waitForBinding === true)
+    await browser.wait(until.textToBePresentInElement(pvcStatus, PVC_STATUS.BOUND));
+};
+
+export const deletePersistentVolumeClaim = async function(name: string, namespace: string) {
+  await goToPersistentVolumeClaims();
+  await selectItemFromDropdown(namespace, namespaceDropdown);
+  await crudView.resourceRowsPresent();
+  await crudView.filterForName(name);
+  await crudView.isLoaded();
+  await crudView.deleteRow('PersistentVolumeClaim')(name);
```

Comment:
Here maybe wait for empty list to make sure the PVC was removed?

Reply:
Doesn't deleteRow function take care of that? https://github.com/openshift/console/blob/master/frontend/integration-tests/views/crud.view.ts#L103

Reply:
Indeed it does, thanks! It wasn't the case sooner.